dfastmi/gui.py

```diff
@@ -582,7 +585,7 @@ def run_analysis() -> None:
     None
     """
     config = get_configuration()
-    if dfastmi.batch.core.check_configuration(rivers, config):
+    if check_configuration(rivers, config):
```

Comment:
lets discuss "dfastmi.batch.core.check_configuration" vs "check_configuration"
and what we would like to adhere to.

Reply:
We keep it like this. Discussed.